webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -68,9 +69,9 @@ class BreadcrumbBar extends React.Component {
           {isSwitchable && (
             <BreadcrumbSwitcher
               open={isSwitcherOpen}
-              isLoadingResources={isLoadingResources}
+              isLoading={isLoadingResources}
               hasError={hasError}
-              resources={resourceSwitcherItems}
+              resources={translateObject(resourceSwitcherItems)}
```

Comment:
`resourceSwitcherItems` is an array of objects, and thats expected by `BreadcrumbSwitcher` it seems.
Now it seems we are translating the whole object If I am reading it correctly.
I guess we eighter need to translate in `BreadcrumbBarActions -> loadSwitcherResourcesByResource -> formatResults` or create new helper `translatePropOfArrayOfObjects(arrayOfObjects, <[props_to_translate]>)`

Reply:
whats wrong with translating the whole object. btw is translating even the desired approach here? isn't it risky/unnecessary? @tbrisker

Reply:
The problems are:

- You are providing array `[{id: .., name: ..},{....}]` to the translations, but the method expects object, and it translates all the values in the object. so in here, it is translating the arrays.
- We don't want to translate anything else from the inner object, just the names.

Reply:
you need to translate each string separately, but not resource names - for example, "compute resource" should be translated but "my-libvirt-CR" should not.